admin-ui/plugins/auth-server/redux/sagas/AuthnSaga.js

```diff
@@ -12,6 +12,11 @@ import {
 import LdapApi from '../../../services/redux/api/LdapApi'
 import ScriptApi from '../../../admin/redux/api/ScriptApi'
 const JansConfigApi = require('jans_config_api')
```

Comment:
Using an incorrect relative path in an import statement is bad because it violates the principle of module integrity by providing a path that does not point to the correct module location, which causes a module resolution failure at build time or runtime.